### ISSUE TITLE
Overhaul Windows documentation

### ISSUE THREAD

**Issue by marosset** [kubernetes/website#31428]
This issue will track a number of improvements SIG-Windows would like to make to the windows documentation.

- Review / update the following pages windows specific pages
  - [ ] https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/
    - [ ] Specify minimum or recommended system requirements 
    - [ ] Split up content into multiple pages
  - [ ] https://kubernetes.io/docs/setup/production-environment/windows/user-guide-windows-containers/
  - [ ] https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/
  - [ ] https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/upgrading-windows-nodes/
  - [ ] https://kubernetes.io/docs/tasks/configure-pod-container/configure-gmsa/
  - [ ] https://kubernetes.io/docs/tasks/configure-pod-container/configure-runasusername/
  - [ ] https://kubernetes.io/docs/tasks/configure-pod-container/create-hostprocess-pod/
  - [ ] https://kubernetes.io/docs/tasks/tools/install-kubectl-windows/

(I'll continue to update this list over the next few days)

/milestone v1.24
/sig windows

**Comment by tengqm** [comment]
Thank you for filing an umbrella issue before working on it.

**Comment by sftim** [comment]
This looks like a partial duplicate of #29769

We should close one of these, I think.
/language en
/remove-kind feature

**Comment by sftim** [comment]
Much of https://kubernetes.io/docs/setup/production-environment/windows/intro-windows-in-kubernetes/ does not belong in the “production environment” section; it's not specific to deploying into production.
We should identify sections of that page to migrate elsewhere.

When moving existing content around, please add the “refactor” label to the PRs - or highlight to a reviewer that this label needs adding. That label helps localization teams spot when apparently-new content is actually being reused.

**Comment by sftim** [comment]
Great to see progress here.

**Comment by sftim** [comment]
@marosset would you be willing to revisit the checkboxes in the issue description and confirm that these are up to date?

**Comment by marosset** [comment]
> @marosset would you be willing to revisit the checkboxes in the issue description and confirm that these are up to date?

Done!

I'll start reviewing the moved content for accuracy now and will either open the PRs myself of find more appropriate PR authors.

**Comment by aravindhp** [comment]
@marosset I have reviewed [configure-runasusername](https://kubernetes.io/docs/tasks/configure-pod-container/configure-runasusername/). I don't see anything missing. I don't see a reason to link to projected volumes and  Windows RunAsUsername [securitycontext-interactions](https://kubernetes.io/docs/concepts/storage/projected-volumes/#windows). Folks who are using projected volumes would read that page and will become aware of the limitation.

**Comment by sftim** [comment]
@marosset, would you consider these docs overhauled?

**Comment by sftim** [comment]
/remove-lifecycle rotten

[Query](https://github.com/kubernetes/website/issues/31428#issuecomment-1350130494) not yet answered. Anyone from SIG Windows is welcome to answer it.

**Comment by sftim** [comment]
How's this work going?

**Comment by aravindhp** [comment]
To kick things off with https://kubernetes.io/docs/tasks/administer-cluster/kubeadm/adding-windows-nodes/ we are going to wait on https://github.com/kubernetes/website/issues/34476#issuecomment-1572577912 that calls out that the Linux sections need to get filled in.

**Comment by aravindhp** [comment]
@tzifudzi will take up `Specify minimum or recommended system requirements`.

**Comment by sftim** [comment]
Also see https://github.com/kubernetes/website/issues/44964

**Comment by neolit123** [comment]
note:
- https://github.com/kubernetes/website/pull/47888

**Comment by sftim** [comment]
What still needs doing here?

**Comment by aravindhp** [comment]
There are some networking pieces that need to be filled in. @adrianm-msft will coordinate.

**Comment by lmktfy** [comment]
What still needs doing here? 

(it's possible that we don't have capacity to maintain current documentation for the Windows support)

**Comment by jrvaldes** [comment]
@zylxjtu can you do a pass and identify what items are still relevant from the description? cc @marosset

**Comment by jrvaldes** [comment]
> Specify minimum or recommended system requirements

https://kubernetes.io/docs/concepts/windows/intro/#windows-hardware-recommendations
